llvm/test/CodeGen/RISCV/rvv/fixed-vectors-llround.ll

```diff
@@ -1,35 +1,28 @@
 ; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py
-; RUN: llc -mtriple=riscv32 -mattr=+v,+f,+d,+zvfh -target-abi=ilp32d \
+; RUN: llc -mtriple=riscv32 -mattr=+v,+f,+d,+zvfhmin,+zvfbfmin -target-abi=ilp32d \
 ; RUN:     -verify-machineinstrs < %s | FileCheck %s --check-prefix=RV32
-; RUN: llc -mtriple=riscv64 -mattr=+v,+f,+d,+zvfh -target-abi=lp64d \
+; RUN: llc -mtriple=riscv64 -mattr=+v,+f,+d,+zvfhmin,+zvfbfmin -target-abi=lp64d \
```

Comment:
Just making a note here that we're switching from zvfh -> zvfhmin. We could add extra RUN lines and test both zvfh + zvfhmin, but I don't think it's that important. Happy to leave this up to you

Reply:
I don't think it's important either, as Zvfh will simply mark f16 as legal, and Zvfhmin is the interesting case.